### PR TITLE
compose: disable custom content

### PR DESCRIPTION
**Pull request by supakeen**

### CODE REVIEW

**Review by ochosi — APPROVED** [review]
LGTM

**Review by thozza — APPROVED** [review]
LGTM

**Review by thozza — DISMISSED** [review]

File: internal/v1/handler_compose_image.go [hunk 1]
```diff
@@ -376,9 +376,16 @@ func buildOSTreeOptions(ostreeOptions *OSTree) *composer.OSTree {
 // validateComposeRequest makes sure the image size is not too large for AWS or Azure
 // It takes into account the requested image size, and the total size of requested
 // filesystem customizations.
+// It also temporarily errors on customrepositories
 func validateComposeRequest(cr *ComposeRequest) error {
 	var totalSize uint64
 	cust := cr.Customizations
+
+	// XXX temporarily disabled
+	if cust.CustomRepositories != nil {
```
Comment (ondrejbudai) [inline internal/v1/handler_compose_image.go:385]: Can you also check len?

**Review by thozza — APPROVED** [review]

**Review by ondrejbudai — APPROVED** [review]

**Review by thozza — APPROVED** [review]
Finally, the tests are also happy \o/

**Pull request merged**